<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadVedioWgtClass</class>
 <widget class="QWidget" name="LoadVedioWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoadVedioWgt</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>视频导入结果</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_VideoFrame">
         <attribute name="title">
          <string>画面帧信息</string>
         </attribute>
         <layout class="QVBoxLayout" name="VLayout_LoadVideo"/>
        </widget>
        <widget class="QWidget" name="tab_Audio">
         <attribute name="title">
          <string>声音信息</string>
         </attribute>
         <layout class="QVBoxLayout" name="VLayout_LoadAudio"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_progressState">
        <property name="title">
         <string>进程状态</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_ffmpegState">
           <property name="text">
            <string>ffmpeg状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ffprobeState">
           <property name="text">
            <string>ffprobe状态：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_selLoadFile">
        <property name="text">
         <string>选择导入文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_fileTagLoadState">
        <property name="text">
         <string>文件头读取状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_fileTagLoadState">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_selFileTag_2">
        <property name="title">
         <string>文件标签</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_file_FileName">
           <property name="text">
            <string>文件名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_file_Path">
           <property name="text">
            <string>文件路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_file_FileType">
           <property name="text">
            <string>文件类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_file_Duration">
           <property name="text">
            <string>持续时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_codec">
           <property name="text">
            <string>视频编码器：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_resolution">
           <property name="text">
            <string>分辨率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_frameRate">
           <property name="text">
            <string>帧率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_BitRate">
           <property name="text">
            <string>视频比特率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_pixFmt">
           <property name="text">
            <string>像素格式：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_video_colorType">
           <property name="text">
            <string>颜色格式：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_audio_codec">
           <property name="text">
            <string>音频编码器：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_audio_ChanelNum">
           <property name="text">
            <string>通道数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_audio_sampleRate">
           <property name="text">
            <string>采样率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_audio_Bitrate">
           <property name="text">
            <string>音频比特率：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="title">
         <string>导入参数</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>帧率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_LoadFrameRate">
             <property name="value">
              <number>30</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>最大宽度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_LoadPicWidth">
             <property name="maximum">
              <number>4080</number>
             </property>
             <property name="value">
              <number>1920</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>最大高度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="SBox_LoadPicHeight">
             <property name="maximum">
              <number>4080</number>
             </property>
             <property name="value">
              <number>1080</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_DecodeAndLoad">
        <property name="text">
         <string>解码并导入视频</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_decodeOnly">
        <property name="text">
         <string>仅解码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_loadOnly">
        <property name="text">
         <string>仅导入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_videoLoadState">
        <property name="text">
         <string>视频读取状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_VideoLoadState">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_audioLoadState">
        <property name="text">
         <string>音频读取状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_audioLoadState">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
